tests/in/scalar-times-matrix.wgsl

```diff
@@ -0,0 +1,11 @@
+[[stage(vertex)]]
```

Comment:
let's not have a separate test file for each operation, instead put it into `operations.wgsl`